pkgs/applications/graphics/tesseract/tesseract5.nix

```diff
@@ -27,13 +29,17 @@ stdenv.mkDerivation rec {
     icu
     pango
     opencl-headers
-  ];
+  ] ++ lib.optionals stdenv.isDarwin (with darwin.apple_sdk.frameworks; [
+    Accelerate
+    CoreGraphics
+    CoreVideo
+  ]);
 
   meta = {
     description = "OCR engine";
     homepage = "https://github.com/tesseract-ocr/tesseract";
     license = lib.licenses.asl20;
     maintainers = with lib.maintainers; [ anselmschueler ];
-    platforms = with lib.platforms; linux ++ darwin;
+    platforms = lib.platforms.linux ++ lib.platforms.darwin;
```

Comment:
referencing just `darwin` here uses the `pkgs.darwin` instead of the platform.

Reply:
```suggestion
    platforms = lib.platforms.unix;
```